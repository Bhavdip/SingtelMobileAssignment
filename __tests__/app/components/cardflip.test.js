import React from 'react';
import { shallow } from 'enzyme';
import { View } from 'react-native';
import CardFlip from '../../../app/components/cardflip';

const setUpComponent = (props = {}) => {
  return shallow(<CardFlip {...props} />);
};

describe('Test CardFlip Component', () => {
  let wrapperComponent = {};
  let props = {
    duration: 800,
    flipZoom: 0.09,
    children: <View />
  };
  beforeEach(() => {
    wrapperComponent = setUpComponent(props);
  });
  it('Test Snapshot Match', () => {
    expect(wrapperComponent).toMatchSnapshot();
  });
});
