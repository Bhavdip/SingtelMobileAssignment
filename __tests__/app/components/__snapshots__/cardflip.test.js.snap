// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CardFlip Component Test Snapshot Match 1`] = `ShallowWrapper {}`;
